CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 1.27.0
+## 1.27.1
+
+* Avoid going exponential on certain recursive `@extend` edge cases.
```

Comment:
Maybe clarify here that "going exponential" refers to the number of selectors generated? On first read, I assumed this just meant exponential in runtime.

Reply:
It's actually the runtime that's the issue here—we do end up trimming most of the selectors, but we do so late enough in the process that we have to process an exponential combination in the meantime.